applications/DotPay.md

```diff
@@ -0,0 +1,298 @@
+
+
+## Team :busts_in_silhouette:
+
+
+### Team members 
+
+We will pleased to offer github accounts, LinkedIn and any other information in private.
+
+All team members can contact privately for any specific information.
+
+* Richard Fang: team leader, core developer
+
+* Fugen Wang: Background development
+  - responsible for the development of the website background.
+
+* Yang Li: 
+  - Responsible for front-end development.
+
+* Peng Qiao: Rust developer 
+  - Responsible for back-end development, relevant development of blockchain, payment task management module, and docking API and key management module on the chain.
+
+ 
+* Wei Zhang: 
+  - Responsible for operation, promotion in the open source community after the website is launched, and attracting open source projects to use our services.
+ 
+* AdaLam:  PD/PMO  
+  - Responsible for product design and project schedule management.
+ 
+ 
+ 
+### Team's Experience
+
+
+* Richard Fang:
+    - As an expert in the field of cloud computing in one of the biggested Internet listed companies with 7 years of rich working  experience. 
+    - The author of a well-known cloud native project which has more than 5K stars and 4k paid enterprise users. 
+
+* Fugen Wang:
+   - CEO of an Internet start-up company and manages more than a dozen employees with 7 years working experience.
+  
+* Yang Li:
+   - Andriod/IOS front-end engineer with 5 years working experience.
+ 
+* Peng Qiao: 
+   - A core member of AI Research Institute wich is one of the top AI listed companies in China with 5 years working experience.
+ 
+* Wei Zhang:
+   - An advertising director, one of the top AI listed companies in China with 7 years of rich working experience.
+ 
+* AdaLam: PD/PMO
+  - Familiar with product design and project schedule management.
+
+
+### Contact
+
+- **Contact Name:**  AdaLam
+- **Contact Email:** adalamlzy@gmail.com
+- **Website:**  comming soon 
+
+- **Contact Name:** Richard Fang
+- **Contact Email:** lamelegdog@gmail.com
+
+### Legal Structure
+
+we will be pleased to offer specific information in private.
+
+
+### Team Code Repos
+
+https://github.com/bytepayment 
+
+https://github.com/bytepayment/bytepay  
+
+
+### Team LinkedIn Profiles
+
+provide in private through Google Form.
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+* **Total Estimated Duration:** 16 weeks
+* **Full-time equivalent (FTE):**  3
+* **Total Costs:**  4,8000 USD
```

Comment:
```suggestion
* **Total Costs:**  48,000 USD
```

Reply:
@semuelle  Thanks for getting back to me so quickly. I feel so sorry about it and it was our unintentional mistake , we've changed it back to the original. 
We also have sent the team details privately through Google Form.We are preparing to add more information about milestone.I'd really appreciate it if you could give me some more suggestions.Thanks again and have a nice day.

Reply:
Thanks

> @semuelle Thank you 



> 
Hi @semuelle  We have added relevant information about Development Roadmap , Ecosystem Fit and Development Status ,and have also submitted the Google Form. could you please have a look and give me some suggestions ? I'd really appreciate it if you could let me know your opinion about it in time.  Looking forward to hearing from you soon.